battle-service/src/main/java/com/adavydenko/tictactoe/battleservice/entities/Step.java

```diff
@@ -0,0 +1,23 @@
+package com.adavydenko.tictactoe.battleservice.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Step {
+    @Id
+    private String id;
+    private Coordinate x;
+    private Coordinate y;
```

Comment:
You probably also need information about who make specific step

Reply:
I thought maybe I can pass an user id separately. And a step relates to a battle, so the user can be found through the battle.